### PR TITLE
Fix set-pipeline

### PR DESCRIPTION
**Pull request by BaiShaoqi**
Now the ORCA_TAG can be automatically extracted from gpdb src.
make set-dev do not comment out the tag_filter
Use right pipeline name to fly -t prod unpause-pipeline

Dev pipeline: https://releng.ci.gpdb.pivotal.io/teams/main/pipelines/greenplum-database-release-fix-make-sbai

### CODE REVIEW

File: concourse/scripts/compile_gpdb_oss.bash [hunk 1]
```diff
@@ -138,6 +138,7 @@ export_gpdb () {
 }
 
 _main () {
+    export ORCA_TAG="$(grep 'ORCA_TAG:' gpdb_src/concourse/tasks/compile_gpdb.yml | cut -d ':' -f 2 | tr -d '[:space:]')"
```
Comment (baotingfang) [inline concourse/scripts/compile_gpdb_oss.bash]: ORCA_TAG=$(grep -Po 'v\d+.\d+.\d+' gpdb_src/depends/conanfile_orca.txt)

Comment (marksliva) [reply concourse/scripts/compile_gpdb_oss.bash]: Here is the commit which bumped the orca to version 3.52.0: https://github.com/greenplum-db/gpdb/commit/b0fb34933d09befd1faa6e22f192005f999e3073

Either way is fine, but `cut/tr` seems to be easier to read and more portable (above grep didn't work on Mac).

File: Makefile [hunk 2]
```diff
@@ -39,13 +39,10 @@ list:
 set-dev: set-pipeline-dev
 
 set-pipeline-dev:
-
-	sed -e 's|tag_filter: *|## tag_filter: |g' concourse/pipelines/gpdb_opensource_release.yml > concourse/pipelines/${PIPELINE_NAME}.yml
```
Comment (marksliva) [inline Makefile]: Maybe this should be in a separate commit. Also, the pattern in 6X-release right now is to `sed` out the `tag_filter` and also not use a secrets variable for `gpdb-git-tag-filter`. So the change can be made in both this pipeline and 6x-release pipeline if needed.

Comment (amilkh) [reply Makefile]: I would approve PR if we don't delete the `sed -e`!

File: concourse/scripts/compile_gpdb_oss.bash (hunk 1 above)
Comment (baotingfang) [reply concourse/scripts/compile_gpdb_oss.bash]: The key point is not the tools, but where do we get the orca version from?

**gpdb_src/concourse/tasks/compile_gpdb.yml**

or 

**gpdb_src/depends/conanfile_orca.txt**

which one file is more reliable？

Comment (amilkh) [reply concourse/scripts/compile_gpdb_oss.bash]: Let's use the tool that works now on cross-platform.
Shaoqi go ahead either way is fine!!

Comment (baotingfang) [reply concourse/scripts/compile_gpdb_oss.bash]: Question: How to get the orca version ever before in gp release?
Ref: https://github.com/greenplum-db/debian-release/blob/master/concourse/scripts/ppa-debian-release.bash#L16

Q: What is the **debian-release** repo?
It publishes the gpdb product to PPA.

Comment (amilkh) [inline concourse/scripts/compile_gpdb_oss.bash]: good comment from Bob! maybe it got deleted accidentally

> Question: How to get the orca version ever before in gp release?
> Ref: https://github.com/greenplum-db/debian-release/blob/master/concourse/scripts/ppa-debian-release.bash#L16
> 
> Q: What is the debian-release repo?
> It publishes the gpdb product to PPA.

File: Makefile [hunk 3]
```diff
@@ -39,13 +39,10 @@ list:
...
-
 	$(FLY_CMD) --target=${CONCOURSE} \
     set-pipeline \
     --pipeline=${PIPELINE_NAME} \
-    --config=concourse/pipelines/${PIPELINE_NAME}.yml \
+    --config=concourse/pipelines/gpdb_opensource_release.yml \
```
Comment (bradfordboyle) [inline Makefile]: Doesn't this make it hard for multiple people to set dev pipelines?

Comment (amilkh) [reply Makefile]: oh but $PIPELINE_NAME can still be overridden, regardless will use same gpdb_opensource_release.yml

File: concourse/scripts/compile_gpdb_oss.bash (hunk 1 above)
Comment (bradfordboyle) [reply concourse/scripts/compile_gpdb_oss.bash]: I know that the PPA release pipeline fetches the orca tag from the conan file but I'm not a big fan of relying on a build-tool file for a build tool that we don't use (conan). The QPA team has an established practice of updating the compile gpdb task yaml.

File: concourse/tasks/compile_gpdb_oss.yml [hunk 4]
```diff
@@ -9,5 +9,3 @@ outputs:
   - name: gpdb_artifacts
 run:
   path: greenplum-database-release_src/concourse/scripts/compile_gpdb_oss.bash
-params:
```
Comment (bradfordboyle) [inline concourse/tasks/compile_gpdb_oss.yml]: Yeah, with this being in a separate repo QPA wouldn't be updating this.

File: Makefile (hunk 2 above)
Comment (BaiShaoqi) [reply Makefile]: 6X-release right now is to sed out the tag_filter, I do not have a strong opinion on this, I deleted this because I did not see why not need tag filter. I can change it back.

File: concourse/scripts/compile_gpdb_oss.bash (hunk 1 above)
Comment (BaiShaoqi) [reply concourse/scripts/compile_gpdb_oss.bash]: Now orca team updates the compile_gpdb.yml whenerver they bumps version , and it seems to me more reasonable to depend on yaml instead of build-tool in this case
